activesupport/lib/active_support/cache/file_store.rb

```diff
@@ -211,17 +211,22 @@ def search_dir(dir, &callback)
         # If the key is not found it is created and set to +amount+.
         def modify_value(name, amount, options)
           file_name = normalize_key(name, options)
+          options = merged_options(options)
+          key = normalize_key(name, options)
```

Comment:
Can reuse `file_name` here.  Or perhaps better: assign `file_name = key` afterward.

Reply:
Hum, no because we use both: `lock_file(file_name)`, `read_entry(key, ...)`

Reply:
By "reuse" I meant set `key = file_name` because they are both `normalize_key(name, options)` (and `normalize_key` [will handle `merge_options` internally](https://github.com/rails/rails/blob/194ce13228f42d441df8f2e78fcee0ebb215e6ef/activesupport/lib/active_support/cache.rb#L940-L957)).

But I actually think it would be nicer to set `file_name = key` afterward:

```ruby
options = merged_options(options)
key = normalize_key(name, options)
version = normalize_version(name, options)
file_name = key
amount = Integer(amount)
```

Reply:
🤦 OK, I think I'm tired tonight. I'll set a reminder to clean this up tomorrow because clearly I'm done for today.